libfreerdp/crypto/tls.c

```diff
@@ -159,6 +159,11 @@ static int bio_rdp_tls_read(BIO* bio, char* buf, int size)
 				break;
 
 			case SSL_ERROR_SYSCALL:
+				if (WSAGetLastError() == WSAEWOULDBLOCK)
```

Comment:
We have seen many patches here, they were all hiding real problems.
Reading errno is not reliable in this case.